net.puddletag.puddletag.yaml

```diff
@@ -0,0 +1,97 @@
+app-id: net.puddletag.puddletag
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+command: puddletag
+rename-desktop-file: puddletag.desktop
+rename-icon: puddletag
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+modules:
+  - name: PyQtApp
```

Comment:
```suggestion
```

This probably trigger a warning with flatpak builder anyway.

Reply:
Right, it was a leftover - fixed.